cmd/agent/app/options/options.go

```diff
@@ -98,6 +98,19 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	))
 	fs.BoolVar(&o.LeaderElection.LeaderElect, "leader-elect", true, "Start a leader election client and gain leadership before executing the main loop. Enable this when running replicated components for high availability.")
 	fs.StringVar(&o.LeaderElection.ResourceNamespace, "leader-elect-resource-namespace", util.NamespaceKarmadaSystem, "The namespace of resource object that is used for locking during leader election.")
+	fs.DurationVar(&o.LeaderElection.LeaseDuration.Duration, "leader-elect-lease-duration", o.LeaderElection.LeaseDuration.Duration, ""+
```

Comment:
Seems the `o.LeaderElection.LeaseDuration.Duration` was not initialized before, which will default to `0`?

Reply:
Same for the below 2 flags.

Reply:
> Seems the o.LeaderElection.LeaseDuration.Duration was not initialized before?
that's right, which should be initialized in function  `NewOptions`

Reply:
Please use `defaultElectionLeaseDuration` as default value, just for readability concern.